xbmc/guilib/GUIBaseContainer.cpp

```diff
@@ -293,7 +293,9 @@ bool CGUIBaseContainer::OnAction(const CAction &action)
 {
   if (action.GetID() >= KEY_ASCII)
   {
-    OnJumpLetter((char)(action.GetID() & 0xff));
+    std::string letter;
+    g_charsetConverter.wToUTF8({action.GetUnicode()}, letter);
+    OnJumpLetter(letter);
```

Comment:
```suggestion
    OnJumpLetter(std::move(letter));
```

Reply:
Do I really need to explicitly move this here?
I thought the compiler would do that in this case.

Reply:
I assumed compilers didn't elide a copy of the last use of a variable before it falls from scope, but I'm probably wrong

Reply:
Mhm, I'm not sure either - maybe someone who knows will answer it.
It's easy to change it to an explicit move, so I'll do that.